recipe/meta.yaml

```diff
@@ -13,22 +13,22 @@ source:
     - no_crt_copy.patch    # [win]
     - no_snprintf.patch    # [win and py35]
     - vs2010_openmp.patch  # [win and py34]
+    - vs2010_path.patch    # [win64 and py34]
 
 build:
-  number: 0
+  number: 1
   features:
     - vc9     # [win and py27]
     - vc10    # [win and py34]
     - vc14    # [win and py35]
-  skip: True  # [win]
 
 requirements:
   build:
     - python  # [win]
 
 test:
   requires:
-    - python {{ environ['PY_VER'] + '*' }}                # [win]
+    - python                                              # [win]
```

Comment:
It works without the version specified? I'm a bit worried about this. Couple options here.

Could try using this

```
test:
    requires:
        - python 2.7.*  # [win and py27]
        - python 3.4.*  # [win and py34]
        - python 3.5.*  # [win and py35]
```

Alternatively, could try unpinning `conda-build`. I believe all the current bugs are fixed. Though don't know if there are new ones or not yet.